src/Roslyn.Diagnostics.Analyzers/Core/DeclarePublicAPIAnalyzer.Impl.cs

```diff
@@ -507,16 +507,34 @@ internal List<ApiLine> GetDeletedApiList()
 
             private bool IsPublicAPI(ISymbol symbol)
             {
+
                 if (symbol is IMethodSymbol methodSymbol && s_ignorableMethodKinds.Contains(methodSymbol.MethodKind))
                 {
                     return false;
                 }
-
                 // We don't consider properties to be public APIs. Instead, property getters and setters
                 // (which are IMethodSymbols) are considered as public APIs.
-                if (symbol is IPropertySymbol)
+                // Unless property is an auto-property.
+                if (symbol is IPropertySymbol propertySymbol)
                 {
-                    return false;
+                    if (symbol.Language != "Visual Basic") return false;
```

Comment:
>"Visual Basic") [](start = 43, length = 15)

Please use the `LanguageNames.VisualBasic` constant: [http://source.roslyn.io/#Microsoft.CodeAnalysis/Symbols/LanguageNames.cs,e8b3edd090bfbe79,references](http://source.roslyn.io/#Microsoft.CodeAnalysis/Symbols/LanguageNames.cs,e8b3edd090bfbe79,references).